src/org/kernelci/util/Job.groovy

```diff
@@ -56,6 +56,12 @@ kci_build \
     def build_env_data = build_env_raw.split('\n').toList()
     def cc_arch = build_env_data[3]
 
+    /* Some architecture builds include other arch */
+    if (cc_arch == 'x86_64')
+        cc_arch = "x86"
+    if (cc_arch == 'riscv')
+        cc_arch = "riscv64"
+
```

Comment:
This should be done in kernelci-core, in the output of `kci_build show_build_env`.

But it doesn't look quite right though, the architecture names have to follow the kernel ones and these have to be `x86_64` and `riscv`.  Could you please explain what the problem is here?

Reply:
As discussed on IRC, we have some jobs scheduled that are pulling non-existing images. Two examples:
```
Init Containers:
  scratch-setup:
    Container ID:  
    Image:         kernelci/staging-clang-16:x86_64-kselftest-kernelci
    Image ID:      
    Port:          <none>
    Host Port:     <none>
    Command:
      /bin/bash
      -x
      -c
    Args:
      mkdir -p /tmp/kci
    State:          Waiting
      Reason:       ImagePullBackOff
```
```
    Image:         kernelci/clang-11:riscv-kselftest-kernelci
    Image ID:      
    Port:          <none>
    Host Port:     <none>
    Command:
      /bin/bash
      -x
      -c
    Args:
      mkdir -p /tmp/kci
    State:          Waiting
      Reason:       ImagePullBackOff
```
There is 2 reasons possible:
1)Some images might include all necessary libraries for both 32 and 64-bit architecture (might be possible with x86 at least, as i know). Then we need to change image name, and use for example for "x86_64" arch - "x86", such as kernelci/staging-clang-16:x86-kselftest-kernelci

2)Some images are missing and need to be built.

Reply:
Ah, so I think we should revert the changes that add Clang architecture images for now so we can take the time to resolve this properly.

Reply:
How come this wasn't detected on staging though?  There should have been plenty of failing builds due to this.

Reply:
This is only related to clang and x86 builds, IMHO it is minor issue and can be fixed trivially, i just need to know correct approach, which is either merge arch name, or add missing docker images.
Thats why i asked @khilman @broonie help on IRC

Reply:
As i understand more correct approach is proper config in build-configs.yaml / build_environments

Reply:
Yes it should just be a few lines of YAML but testing it on staging will take time.  So if we can do the production update today by reverting 2 commits on the `kernelci.org` branch it seems like the best approach, when we'll have the whole week to get this resolved properly.